src/Mutant/Mutant.php

```diff
@@ -54,7 +55,7 @@ public function __construct(string $mutatedFilePath, Mutation $mutation, string
         $this->diff = $diff;
     }
 
-    public function getMutatedFilePath(): string
+    public function getMutantFilePath(): string
     {
         return $this->mutatedFilePath;
```

Comment:
let's rename a property as well to make it consistent

Reply:
👍